src/index.css

```diff
@@ -7,13 +7,17 @@ if there's any unexpected styling (esp. coloring), check node_modules\bootstrap\
 
 :root{
   --dullblue: #465491;
+  --white: #FFFFFF;
 
   /*for hover transparency transitions*/
-  --dullblueRGBA1: #46549100;
-  --dullblueRGBA2: #465491FF;
+  
+  /*ie11 does not support hex rgba*/
+  --dullblueRGBA1: rgba(70,84,145,0);
+  --dullblueRGBA2: rgba(70,84,145,1);
 
-  --whiteRGBA1: #FFFFFF00;
-  --whiteRGBA2: #FFFFFFFF;
+  /*#FFFFFF*/
+  --whiteRGBA1: rgba(255,255,255,0);
+  --whiteRGBA2: rgba(255,255,255,1);
```

Comment:
I'd suggest using SASS plus their mixins instead of having 2 vars like this -- especially for white
SASS allows for `rgba(white, 0)` which you could just use inline